src/ui/components/analytics-view.ts

```
@@ -0,0 +1,330 @@
+    // Filter out items with zero or positive consumption for charts to make them cleaner
+    const consumedItemsForChart = consumptionReport.filter(c => c.consumedVolumeMl !== undefined && c.consumedVolumeMl > 0);
+
+    const productNames = consumedItemsForChart.map(c => loadedProducts.find(p => p.id === c.productId)?.name || 'Unbekannt');
+    const consumptionValuesMl = consumedItemsForChart.map(c => c.consumedVolumeMl || 0);
+    const costValues = consumedItemsForChart.map(c => c.costOfConsumption);
+
+    renderCharts(productNames, consumptionValuesMl, costValues);
+
+    // Display summary
+    const summaryContainer = document.getElementById('report-summary')!;
+    const totalConsumedItems = consumptionReport.reduce((sum, item) => sum + (item.consumedUnits > 0 ? 1 : 0) ,0)
```

Biome (1.9.4)

[error] Shouldn't redeclare 'totalConsumedItems'. Consider to delete it or rename it.

'totalConsumedItems' is defined here:


(lint/suspicious/noRedeclare)